control_plane/roles/webui_awx/files/awx_projects_pvclaim.yml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
```

Comment:
Could we merge the PV description (`control_plane/roles/webui_awx/files/awx_projects_pv.yml`) together with the claim into a single file?

Reply:
Yes.. I have merged them into a single file